projects/plugins/jetpack/_inc/client/writing/composing.jsx

```diff
@@ -152,6 +154,32 @@ export class Composing extends React.Component {
 						</ModuleToggle>
 					</FormFieldset>
 				</SettingsGroup>
+			),
+			blocks = (
+				<>
+					<SettingsGroup
+						support={ {
+							text: __(
+								'Jetpack includes some blocks which can help you create your pages exactly the way you want them.',
+								'jetpack'
+							),
+							link: getRedirectUrl( 'jetpack-support-blocks' ),
+						} }
+					>
+						<p className="jp-settings-card__blocks-description">
+							{ __(
+								'Jetpack blocks give you the power to deliver quality content that hook website visitors without needing to hire a developer or learn a single line of code.',
+								'jetpack'
```

Comment:
```suggestion
								'Jetpack blocks give you the power to deliver quality content that hooks website visitors without needing to hire a developer or learn a single line of code.',
								'jetpack'
```
Same suggestion as in #23468.